docs/workflow/continuous_integration.rst

```diff
@@ -0,0 +1,203 @@
+******************
```

Comment:
You need to increase the no of `*` in this and line 3 - upto (or even more) the length of text in line-2, otherwise there will be problem while Sphinx render this RST in HTML. 

Remember always this as a rule of thumb, length of these underlines (or overlines) should be >= length of text for which you're using them.